tests/login.go

```diff
@@ -104,6 +104,17 @@ func LoggedInAlpineExpecter(vmi *v1.VirtualMachineInstance) (expect.Expecter, er
 // LoggedInFedoraExpecter return prepared and ready to use console expecter for
 // Fedora test VM
 func LoggedInFedoraExpecter(vmi *v1.VirtualMachineInstance) (expect.Expecter, error) {
+	virtClient, err := kubecli.GetKubevirtClient()
+	expecter, err := LoggedInFedoraExpecterWithoutIPv6Address(vmi)
+	if err != nil {
+		return expecter, err
+	}
+	return expecter, configureIPv6OnVMI(vmi, expecter, virtClient)
+}
+
+// LoggedInFedoraExpecter return prepared and ready to use console expecter for
+// Fedora test VM
+func LoggedInFedoraExpecterWithoutIPv6Address(vmi *v1.VirtualMachineInstance) (expect.Expecter, error) {
```

Comment:
Yep I am ok, but I want to make this retro compatible and don't want to change All the others... another option I was thinking is create a `liblogin` or `libconsole` with this function and it will be `liblogin.LoggedInFedoraExpecter` and make the one a utils.go call `liblogin.LoggedInFedoraExpecter` and then configure ipv6, what do you think ?